lib/galaxy_test/api/test_histories.py

```diff
@@ -92,6 +92,15 @@ def test_index_order(self):
         assert index_response[0]["id"] == newer_history_id
         assert index_response[1]["id"] == slightly_older_history_id
 
+    def test_index_query(self):
+        expected_history_name = "TestHistoryThatMatchQuery"
+        self._create_history(expected_history_name)["id"]
+        self._create_history("TestHistoryThatDoesNotMatchQuery")
+        query = f"?q=name&qv={expected_history_name}"
```

Comment:
Can you also add a test for 2 q/qv pairs?
In particular, I would be interested in checking that `q=deleted&qv=True` works.

Reply:
Sure!